pagopa/io-app-design-system
# [IOAPPX-254] Add iOS smooth corners to all rounded rectangles

## Short description
This PR adds iOS smooth corner to all rounded rectangles. To learn more: 🔗 [Figma Learn · Corner smoothing for squircles](https://help.figma.com/hc/en-us/articles/360050986854-Adjust-corner-radius-and-smoothing#:~:text=Corner%20smoothing%20for%20squircles,-Designers%20noticed%20a&text=This%20corner%20smoothing%20created%20a,Apple's%20later%20designs%2C%20including%20OS14.). This `View` style property is only supported on iOS 13 (and later) and by RN `0.72` (and later), but should be gracefully ignored.

## List of changes proposed in this pull request
- Set `borderCurve` to `continuous`

## How to test
We don't use high radius values in most of our new components, so you shouldn't notice anything
